src/asm/parser.y

```diff
@@ -430,12 +432,15 @@ enum {
 %type	<nConstValue>	sectorg
 %type	<sectSpec>	sectattrs
 
+%type	<inlineFragmentName>	inline_fragment
```

Comment:
Since all uses expect a symbol name, shouldn't the return type be `tzSym`?

Reply:
Sure. It used to be a custom struct for storing the symbol name along with backing up `nPCOffset` and `nListCountEmpty`, but now those were eliminated.